pkg/logstash.default

```diff
@@ -38,4 +38,4 @@
 # If this is set to 1, then when `stop` is called, if the process has
 # not exited within a reasonable time, SIGKILL will be sent next.
 # The default behavior is to simply log a message "program stop failed; still running"
-KILL_ON_STOP_TIMEOUT=0
+#LS_KILL_ON_STOP_TIMEOUT=0
```

Comment:
This is introducing a breaking change here, is there any reason not to keep the name for now? I agree the rename is better, but if possible not to break the var usage, we should keep this as the same. Changing names will be good for 5.0 release.

I mean this option is working for stop function, so we should somehow keep it like this.